vmicore/rust_src/protos/pkg/logging/api/field.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+option go_package = "bitbucket.gdata.de/ps/hive-operations/pkg/logging/api";
```

Comment:
nit: replace this probably with https://docs.buf.build/generate/managed-mode#go_package_prefix

Reply:
Thanks for pointing this out, very interesting. But I think, for now I'll just remove the `go_package` option since we're still not sure how to coordinate the synchronization of protobuf definitions with other services. @cavus700 had the idea to host them on https://buf.build/ for example.

Reply:
the first part is exactly what I meant :smile: the link I added describes how the `go_package` can be set with buf without requiring it to be present in the protobuf file.

Using the buf registry is one option fur sure (even though the pricing is not yet clear in the future) I played around with it already with INetMock and it at least for Go it's working already flawlessly. I think it's a good idea to give it at least a shot :blush:

Reply:
Sorry, I think I phrased that confusingly. I meant I won't create a `buf.gen.yaml` just yet since we need to figure out the hosting part first. The pricing thing is of course a bit concerning, but I guess we'll have a look in the near future.

Reply:
This essentially means that right now we have two versions of those protobuf files, one here, one in our hive-operations repo, which just happen to have the same definitions. Not ideal, I know, but something I'd rather improve in the future instead of doing it now. Could even be tomorrow, depending on how much time @cavus700 has to spare...

Reply:
I guess my comment was also somehow misleading :smile: you don't need a `buf.gen.yaml` in this repo no matter if you're going to use the buf registry or any other solution. You might want to add a `buf.yaml` at some point in the future to be able to use `buf lint` in this repo but that's of course completely up to you.

To be able to generate the required Go code in the hive-operations repo it would even be enough to clone this repo to a temp directory generate the required Go code and delete the cloned repo afterwards if you'd decide to go without the buf registry and don't want to keep the protobuf files in sync manually but I'm pretty sure you'll figure out a feasible solution as soon as you have time to actually look into it :blush: